goutubedl.go

```diff
@@ -213,6 +213,10 @@ type Options struct {
 	HTTPClient        *http.Client                  // Client for download thumbnail and subtitles (nil use http.DefaultClient)
 	MergeOutputFormat string                        // --merge-output-format
 	SortingFormat     string                        // --format-sort
+
+	// Set to true if you don't want to use the result.Info structure after the goutubedl.New() call,
+	// so the given URL will be downloaded in a single pass in the Download() call.
+	NoInfoDownload bool
```

Comment:
Now when there is a package `Download` function maybe this does not need to be exported? i'm thinking that maybe how this is done internally might be refactored in the future using some help/aux method instead of using a struct member. Then it would be good if this is not exported at all to not break the public API.